### PR TITLE
Validate Eggdog

### PR DESCRIPTION
**Pull request by fauzaanchan**
# Validate [EGG](https://solscan.io/token/EXA537HSBVpsFijENbt6Muuy9AADUN8dUmYKD4oKbjJE)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/eggdogsolana/status/1739391311973929297
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/eggdog

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by frqntr** [comment]
Let's make $EGG GROW WITH VALIDATE IT

File: validated-tokens.csv [hunk 1]
```diff
@@ -1076,3 +1076,1082 @@ Yaku,YAKU,NGK3iHqqQkyRZUj4uhJDQqEyKKcZ7mdawWpqwMffM3s,9,https://raw.githubuserco
...
+LOAF CAT,LOAF,3de2yRhtD4VbJBb8EQAQffYMPLU4EnSHT1eveBwiL3tn,6,https://loaf.cat/img/LOAFLOGO.png,true
+Spurdo Spärde,spurdo,8HJ81sHVsmJMzm6XBfTgywWQXwQTmVdmXaQHm7htGPH2,9,https://bafkreiedrxet7nlt4bcqwkf6eeouhp3r2ioswagyn7dpb4eqao4lwweede.ipfs.nftstorage.link,true
+LessFnGas,LFG,LFG1ezantSY2LPX8jRz2qa31pPEhpwN9msFDzZw4T9Q,7,https://mogos7pjgqvbf3z7kzixpv67uxngkrifdfoddttz2tu6ywphk33q.arweave.net/Y4zpfek0KhLvP1ZRd9ffpdplRQUZXDHOedTp7FnnVvc?ext=png, true 
+Yaku,YAKU,NGK3iHqqQkyRZUj4uhJDQqEyKKcZ7mdawWpqwMffM3s,9,https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EK58dp4mxsKwwuySWQW826i3fwcvUK69jPph22VUcd2H/logo.png,true
+BURRRD,BURRRD,F8qtcT3qnwQ24CHksuRrSELtm5k9ob8j64xAzj3JjsMs,4,https://arweave.net/5qQw9rLaDDQeFJ6-384p8yRmnl_8zK47JBwl3QYmD60,true
+Andrea Von Speed,VONSPEED,DVzrCErBzydh92bBzSJX1dKwVvb4omwhrvNz8CwRyxxV,6,https://i.ibb.co/bmZ6pZP/8338-C1-BC-0745-4201-BE73-D39-ADCCE129-F.jpg,true
+Speero,Speero,C7K4Tot6fnnNwhWpqw9H277QPcP56vHAEeXubRHDyCo9,6,https://assets.coingecko.com/coins/images/34252/standard/speeroLOGO.png,true
+Eggdog,EGG,EXA537HSBVpsFijENbt6Muuy9AADUN8dUmYKD4oKbjJE,9,https://i.ibb.co/ZXQdDdd/eggdog-logo.png,true
```
Comment (fauzaanchan) [inline validated-tokens.csv:2157]: Validate Eggdog

**Review by fauzaanchan — COMMENTED** [review]
Validate Eggdog

File: validated-tokens.csv (hunk 1 above)
Comment (eggdogsolana) [reply validated-tokens.csv:2157]: Validate Eggdog

**Comment by Jymfletzfur** [comment]
duplicate ticker exists on strictlist

**Pull request closed without merging**